subsys/net/ip/l2/openthread/Kconfig

```diff
@@ -0,0 +1,137 @@
+# Kconfig - OpenThread driver configuration options
+#
+#
+# Copyright (c) 2017 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+#
+# OpenThread options
+#
+menuconfig NET_L2_OPENTHREAD
+	bool
+	prompt "OpenThread L2"
+	depends on FLASH
```

Comment:
Using "depends on" is better than "select" but isn't it a bit cumbersome here. I mean that if FLASH is not selected in some other part of the system, then this option will not be seen in menuconfig for example. I am ok with this thou, just a note.